server/src/main/resources/log4j.properties

```diff
@@ -15,7 +15,7 @@
 #
 
 # Set root logger level to DEBUG and its only appender to A1.
-log4j.rootLogger=INFO, A1
+log4j.rootLogger=DEBUG, A1
```

Comment:
It's very preferred to save our current logging format https://github.com/mendersoftware/conductor/blob/mender/master/server/src/main/resources/log4j.properties#L25
```
log4j.appender.A1.layout.ConversionPattern=[%d{ISO8601}] [%t] %-5p %c %x - %m%n
```
and "INFO" log level as well

Reply:
after looking at this closely, I think we should do this (configuration change) in the other way than changing the default configuration in the conductor source code
e.g. we can provide our configuration file

Reply:
Well, it's a good question, because if we do it with files then we need to provide those files everywhere in addition to an image, which is not the best way of doing things with containers. I thought we have separate mender/master brach exactly for this purpose.

Reply:
I think once I did do it with environment variables:

- http://logging.apache.org/log4j/2.x/manual/lookups.html ("Environment Lookup")

- http://johnjianfang.blogspot.com/2009/07/how-to-pass-environment-variable-to.html

but it has been a while, and I have not tested this recently. log4j 2.x is required.

Reply:
I didn't know we are using mender/master for building conductor images...

Reply:
the first reason to have a fork of the conductor was that netflix is not providing conductor images so we needed to build them somehow

Reply:
https://hub.docker.com/r/devopsopen/docker-netflix-conductor/tags

Reply:
and who is this devopsopen?

Reply:
yeah, I mis read the login name as one of the conductor authors. oh well. we need to make order with those conductor images anyway. perhaps it is a good time to do it now.

Reply:
about configuration: we are already providing config file for the conductor:
https://github.com/mendersoftware/integration/blob/c1673864a6eb662825ca64e20413054c6cfd1b01/template/prod.yml#L124
https://github.com/mendersoftware/integration/tree/master/conductor/server/config